docs/Using-Virtual-Environment.md

```diff
@@ -0,0 +1,52 @@
+# Using Virtual Environment
+
+## What is a Virtual Environment?
+A Virtual Environment is a self contained directory tree that contains a Python installation 
```

Comment:
Maybe we should link to a Wiki page for more info on virtual env? Or a more helpful "Overview" link.

Reply:
How about adding the actual documentation to python virtual environment rather than us writing an overview. What do you think of this?

"A Virtual Environment is a self contained directory tree that contains a Python installation 
for a particular version of Python, plus a number of additional packages. To learn more about 
Virtual Environments see [here](https://docs.python.org/3/library/venv.html)"